tests/test_0520-dynamic-classes-cant-be-abc-subclasses.py

```diff
@@ -1,16 +1,15 @@
 # BSD 3-Clause License; see https://github.com/scikit-hep/uproot4/blob/main/LICENSE
 
 import pickle
+import sys
 
 import numpy as np
 import pytest
 import skhep_testdata
 
-import uproot
-
 
 @pytest.mark.skipif(
-    uproot._util.py36,
+    sys.version_info < (3, 7),
```

Comment:
I dropped this one helper; the sys.version_info is much easier to handle statically (pyupgrade can remove branches when it uses this syntax, for example), reads better (you don't have to look up the meaning of py36), and was the only place this helper was used.

Reply:
There was a pack of `py26`, `py27`, `py35`, `py36` booleans, as well as an `iswin`. The `py36` was only needed in one case, which is why I still consider Python 3.6 acceptable—it needs very few work-arounds.

Meanwhile, there are a lot of different ways of saying `sys.version_info[0] == 3 and sys.version_info[1] < 7`, etc. (which mean subtly different things). This tuple-inequality idiom is succinct, and if it's recognized by static checkers, then it's the one to use. But I didn't know that there was a "one to use."

Similarly for the Windows-check: `iswin` has been redefined a few times as some ways of checking platform haven't been as robust as the one we have there now. By putting it in a boolean, we could fix it centrally.

Another one is `isstr`, which was a lot more important when we had to deal with different meanings of `str`, `bytes`, and `unicode`.

The `isint` is one that I think should stay. There isn't a good idiom for "integer but not boolean." There was some version of NumPy where NumPy scalars weren't being recognized as integers (can't remember which, exactly) and anyway Python and NumPy differ on whether booleans should be considered integers. Having a central facility for that makes it possible to control from one location.